server/lib/ReportManager.js

```diff
@@ -97,6 +98,13 @@ class ReportManager {
       throw new BadRequest(`The application does not support rendering ${options.renderer}.`, 'ERRORS.INVALID_RENDERER');
     }
 
+    // @TODO user information could be determined by report manager, removing the need for this check
+    if (this.options.saveReport && !this.options.user) {
+      let invalidSaveDescription = 'Report cannot be saved without providing a `user` entity to ReportManager';
+      console.error(invalidSaveDescription);
+      throw new InternalServerError(invalidSaveDescription);
```

Comment:
The `console.error` is needed. The question is if we can use `console.error` in combination with `throw new ...`